entrypoint.sh

```diff
@@ -3,4 +3,8 @@
 set -euo pipefail
 
 ./prepare.sh
-/usr/bin/bootc-image-builder --store /store --rpmmd /rpmmd --output /output  "$@"
+tmpout="$(mktemp -p / -d)"
+/usr/bin/bootc-image-builder --store /store --rpmmd /rpmmd --output "${tmpout}" "$@"
+echo "Moving artifacts to output directory."
+cp -rf "${tmpout}/." /output
```

Comment:
Doesn't this still end up with the artifact owned by root? Since root is doing the cp?

Reply:
It should at least ensure that the file is world readable.

Reply:
Yeah, we can easily test this by doing a test branch with this one here and https://github.com/osbuild/bootc-image-builder/pull/33 and removing the workarounds (i.e. https://github.com/osbuild/bootc-image-builder/pull/33/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR103) - I can do this later today or in my morning if desired

Reply:
The problem is on a Mac when you mount the homedir into the MAC, it can only store files owned by the host user. Imaging the User UID=1000. When the user runs the VM, it is running as UID=1000.  When the user picks a directory for /output that is in their homedir, then bootc-image-builder attempts to create a UID==0 file in the users homedir, which ends up with the VM process (qemu) running as UID=1000 attempting to create a file UID=0 on the host system. The MAC Os does not allow this.  

On Windows this does not cause issues because the UIDs/GIDs mean nothing on Windows.
On Linux the qemu process is running within the User Namespace, so that UID=0 within the processes User namespace ends up being UID=1000 in the users homedir.

So this is only an issue on Macs.